prometheus/value.go

```diff
@@ -46,6 +46,9 @@ type value struct {
 	// operations.  http://golang.org/pkg/sync/atomic/#pkg-note-BUG
 	valBits uint64
 
+	// valInt is used to store values that are exact integers
```

Comment:
Let's not have a blank line before this line. valBits and valInt are kind of related.